recipes/fastcluster/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "fastcluster" %}
+{% set version = "1.1.21" %}
+{% set sha256 = "c0f7c234a93d5414e9b1300b17ec5c82bd9a9e4c210fc5d9bc3e2bc82fced989" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+  run:
+    - python
+    - numpy x.x
+
+test:
+  imports:
+    - fastcluster
+
+about:
+  home: http://danifold.net/
```

Comment:
This looks like a link to a personal page, rather than the home page of the project. Maybe the following instead?

http://danifold.net/fastcluster.html